_includes/video.html

```diff
@@ -0,0 +1,11 @@
+<div class="video">
+  <div class="video__left">
```

Comment:
Nitpicky, maar misschien beter om deze te hernoemen naar `video__description` en de andere naar `video__player` of iets dergelijks? Aangezien met rtl rechts links is, en andersom :)